packages/@o3r/eslint-plugin/src/rules/yaml/yarnrc-package-extensions-harmonize/yarnrc-package-extensions-harmonize.spec.ts

```diff
@@ -1,24 +1,19 @@
+import * as fs from 'node:fs/promises';
 import * as path from 'node:path';
-import {
-  cleanVirtualFileSystem,
-  useVirtualFileSystem
-} from '@o3r/test-helpers';
 import {
   RuleTester
 } from '@typescript-eslint/rule-tester';
-import yamlParser from 'yaml-eslint-parser';
+import * as yamlParser from 'yaml-eslint-parser';
 import yamlDependencyVersionsHarmonize from './yarnrc-package-extensions-harmonize';
 
-const virtualFileSystem = useVirtualFileSystem();
-
 const ruleTester = new RuleTester({
   languageOptions: {
     parser: yamlParser,
     parserOptions: {
       defaultYAMLVersion: '1.2'
     }
   }
-});
+} as any);
```

Comment:
I don't understand why the any is needed here... but without it's failing because it says that `languageOptions` does not exist on `FlatConfig.Config` (which is false, it does)